app/forms/employers_contract_claim.rb

```diff
@@ -0,0 +1,5 @@
+class EmployersContractClaim < BaseForm
+  attribute :make_employer_contract_claim, :boolean
+  attribute :claim_information, :text
+  attribute :uploaded_content, :string # or binary?
```

Comment:
Don't worry about this right now - we would not store the contents of the file in the database anyway - it would always be a reference (URL) on S3 or something.  But we will address this when we get around to persistence

Reply:
Ok thanks. That sounds like it'll be a string anyway so I'll remove the binary comment.